lib/utils.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+const appRoot = require('app-root-path');
+
+const root = dir => appRoot.resolve(dir || '');
+
+/**
+ *
+ * @param filePath
+ * @returns {*} json formatted content
+ */
+const loadYaml = filePath => {
+  try {
+    return yaml.safeLoad(fs.readFileSync(filePath), 'utf8');
+  } catch (e) {
+    return false;
+  }
+};
+
+/**
+ * get server side configuration
+ *
+ * @returns {*|{}}
+ */
+const getServerConfig = () => {
+  // parse config yaml
+  let config = loadYaml(root('server/config.yml')) || {};
+  let tryFile = root('server/local_config.yml');
+  if (fs.existsSync(tryFile)) {
+    // merge local_config
+    let local_config = loadYaml(tryFile);
+    if (typeof local_config === 'object') {
+      Object.assign(config, local_config);
+    }
+  }
+  return config;
+};
+
+module.exports = {
+  root,
+  loadYaml,
```

Comment:
seems like ``loadYaml `` do not need to be exported

Reply:
At here,` loadYaml` seems no need to exported, but all functions in utils label as `exported` is not bad